src/main/steps/tasklistresponse/proceedings/document-summary/template.njk

```diff
@@ -3,17 +3,18 @@
  {% from "govuk/components/table/macro.njk" import govukTable %}
 
  {% block page_content %}
-   {% set html %}
-   <h3 class="govuk-notification-banner__heading">
-     {{headingTitle}}
-   </h3>
-
-   {% endset %}
-
-   {{ govukNotificationBanner({
-   html:  html,
-   type: "success"
- }) }}
+ <div class="govuk-notification-banner govuk-notification-banner--success govuk-!-margin-bottom-29" role="alert" aria-labelledby="govuk-notification-banner-title" data-module="govuk-notification-banner">
```

Comment:
Test and let me know if this works.

Reply:
Hugh has done this change. I have tested this change.. working fine.